plugin/pkg/scheduler/scheduler.go

```diff
@@ -89,6 +92,7 @@ func (s *Scheduler) Run() {
 
 func (s *Scheduler) scheduleOne() {
 	pod := s.config.NextPod()
+	s.config.Modeler.ForgetPod(pod)
```

Comment:
Can you add a comment here? You don't need to explain the whole thing you explained to me in person. Just saying something like "Pod shouldn't be in assumed if we are about to schedule it" would be fine.